pkg/render/intrusion_detection.go

```diff
@@ -729,14 +759,62 @@ func (c *intrusionDetectionComponent) deploymentPodTemplate() *corev1.PodTemplat
 			ServiceAccountName: IntrusionDetectionName,
 			ImagePullSecrets:   ps,
 			InitContainers:     initContainers,
-			Containers: []corev1.Container{
-				container,
-			},
-			Volumes: volumes,
+			Containers:         containers,
+			Volumes:            volumes,
 		},
 	}, c.cfg.ESClusterConfig, c.cfg.ESSecrets).(*corev1.PodTemplateSpec)
 }
 
+func (c *intrusionDetectionComponent) deployWebhooksController() bool {
+	// deploy webhooks controller container only for:
+	return c.cfg.ManagedCluster || // managed clusters or ...
```

Comment:
This should also be one line and the comment should be written out in its entirety above the return statement. At least in this codebase, but also more generally, we usually avoid comments at the end of lines.

This can also be simplified to this:
```go
c.cfg.ManagedCluster || !c.cfg.ManagementCluster
```

Since the `!c.cfg.ManagedCluster` is implied due to the `||` condition.

Reply:
Good point, changed.